src/utils/utilityFunctions.ts

```diff
@@ -77,3 +77,36 @@ export const isURL = (str: string) => {
   // test the string against the regular expression
   return urlRegex.test(str);
 };
+
+export const isValidPhone=(num:string):boolean=>{
```

Comment:
this is not valid way of checking card is valid or not.Please take reference of this blog : 
https://learnersbucket.com/examples/javascript/credit-card-validation-in-javascript/#:~:text=Validating%20credit%20card%20in%20javascript%20with%20Luhn's%20algorithm.&text=Get%20a%20double%20of%20every,divisible%20then%20it%20is%20valid.